heapster-saw/examples/Makefile

```diff
@@ -39,7 +39,7 @@ ifeq ($(CI),)
   dilithium: dilithium.patch
 	rm -rf dilithium
 	git clone https://github.com/pq-crystals/dilithium.git
-	cd dilithium && git checkout standard
+	cd dilithium && git checkout 918af1a6eaedcedf9fdd8aaaca6c1fccd5a7a51f
```

Comment:
Worth mentioning that if you update the commit hash, you should also update the commit hash in `Dilithium2.saw`.